### DISCUSSION TITLE
How to translate these?

### DISCUSSION THREAD

**Discussion by ImaCrea** [thuliteio/doks#575] [Q&A] [answered]
Hi there,
the i18n built in feature is amazing, thanks.

For the moment, I'm starting with one language so multi languages is not enabled. But I need to have the full site in French. I'm using the **child** version of the theme and it's ok for 99% of the website.

Still, there are two things in the interface that I'm not able to translate. I'm wondering if it's possible or if I need to custom the *parent* version?

![Capture d’écran 2021-12-13 à 11 06 32](https://user-images.githubusercontent.com/764332/145793818-c0be1cba-e399-4236-be5e-5dc6b192c3a9.png)

Thanks for your help :)

**Comment by h-enk** [comment, marked as answer]
Unfortunately, these strings are still hardcoded (will be fixed in an upcoming release of Doks).

You can do this by overwriting the relevant parent files (that's how the child theme works),

So, in your case:

-  Copy `./node_modules/@hyas/doks/layouts/partials/header/header.html` to `./layouts/partials/header/header.html`, and update line 102. Result:

    ![Snag_12bf6766](https://user-images.githubusercontent.com/3902872/145798134-5d01287b-e00e-4fac-8f32-e4cf47b165b8.png)

- Copy `./node_modules/@hyas/doks/layouts/partials/sidebar/docs-toc.html` to `./layouts/partials/sidebar/docs-toc.html`, and update line  3. Result:

    ![Snag_12c071d1](https://user-images.githubusercontent.com/3902872/145798289-05c2bb5c-8ae9-4537-bc57-9b90f0bf804b.png)

**Reply by ImaCrea** [reply]
Hello, thank you for your quick reply.

I tried the first suggestion but it doesn't seem to change. Looks like the original header.html in node_modules keeps the priority. I tried to delete it but then it breaks the website and can't finish building because of it missing.

I'm using Docker-compose. I did `docker-compose down` and then `docker-compose up` again but doesn't solve either.

Another idea maybe please?

**Reply by h-enk** [reply]
Try clearing the cache?

**Reply by h-enk** [reply]
The Hugo and/or Docker cache I mean

**Reply by ImaCrea** [reply]
I tried your suggestion on the Doks' help : `npm run clean` but still not seeing any change. 🤔

**Reply by ImaCrea** [reply]
ok sorry ! my mistake, I had a typo in the folder "header" in my layout. it's all good. No need to refresh cache or anything. sorry for bothering and thank you again for your help! 😅
